Made changes to the animation as suggested :)

Capt-Titanium opened the pull request:
My changes were not reflecting in the old PR so I had to close it :/
@MissMeg please check the animations now...

MissMeg commented:
Hi @Capt-Titanium I took a look at your PR. It looks like the same animations that were in your previous PR. I didn't see the changes that were discussed:

> Right now, the number of animations and the use of multiple different animations make the site kind of look like a retro slideshow presentation. The todo and grocery list are also not animated at the moment so they stick out when the page is loaded. Here's a great resource on using animations that I like to reference.

>Let's do this:
>Keep the header animation for the date
>For the rest of the page, have it all fade in after the title to keep it simple

Capt-Titanium commented:
@MissMeg  here's what I changed in the animations 

- Keep the header animation for the date : I did keep it.
- For the rest of the page, have it all fade in after the title to keep it simple : removed all other animations like rotate or fade in left and kept everything to just fade in just after the title animation finishes.

I noticed that I had not changed the rounded buttons in the bottom so I changed them in the PR where I made the site responsive. 

Here is a link to the video of how it looks on my screen 
https://fluvid.com/videos/detail/Ay_EhvzeRCmaMYqx

If this is not how you wanted the animation then please give more details of how you want the animation.

MissMeg commented:
I cannot see the video you shared, it is private. 

[Here's what it looks like on my screen](https://drive.google.com/file/d/1cjhHIpIE34Yr9Ad3OjmYJ-5cS8Kl4Otz/view?usp=sharing). It seems like the same animations are there.

Capt-Titanium commented:
@MissMeg seems like it isn't showing the changes in your machine for some reason. I did change the animations and the changes reflected on my machine perfectly. 

[Here's what it looks like in my machine](https://drive.google.com/file/d/1axtx3zQ5W2mzKdObT7OB2fm4wNjnLqer/view?usp=sharing)

MissMeg commented on templates/index.html:
```diff
@@ -29,18 +37,18 @@ <h1 class="col">{{ date }}</h1>
                                     <div class="row">
-                                        <p class="col"><i class="today-weather fas fa-tint"></i>60%</p>
-                                        <p id="wind" class="col"><i class="today-weather fas fa-wind"></i>12mph</p>
+                                        <p class="col"><i class="today-weather fas fa-tint wow fadeInLeft" data-wow-delay="0.8s"></i><span class="wow fadeInLeft" data-wow-delay="0.7s">60%</span></p>
```
fadeInLeft old animation

MissMeg commented on templates/index.html:
```diff
@@ -52,7 +60,7 @@ <h1 class="col">{{ date }}</h1>
                             {% for event in events %}
                                 <div class="col">
-                                    <div class="card mb-3" style="max-width: 540px;">
+                                    <div class="card mb-3 wow flipInX" data-wow-delay="0.9s"" style="max-width: 540px;">
```
flipInX old animation

MissMeg commented:
The changes are not pushed to this PR then because I am running the code from this PR. You can see in the [files changed](https://github.com/MissMeg/home-automation-app/pull/34/files) that the HTML file in this PR still has the old animations. I made note of two instances above ^^

Capt-Titanium commented:
@MissMeg I observed that even though I committed new changes in the webpage they didn't show up in my forked repo. Sorry, for the inconvenience. I hope the issue will be resolved now.

MissMeg commented:
There it is 😄 No worries! Looks good 👍
